packages/actor-query-operation-quadpattern/test/ActorQueryOperationQuadpattern-test.ts

```diff
@@ -52,9 +52,11 @@ describe('ActorQueryOperationQuadpattern', () => {
     let mediator;
     let metadata;
     let metadataRaw;
+    let metadataContent;
 
     beforeEach(() => {
-      metadataRaw = Promise.resolve({ totalItems: 3 });
+      metadataContent = { totalitems: 3};
+      metadataRaw = Promise.resolve(metadataContent);
```

Comment:
`metadataRaw` is not being used elsewhere anymore, I would merge it with the following line as follows: `() => Promise.resolve(metadataContent)`